sdk/communication/azure-communication-jobrouter/src/main/java/com/azure/communication/jobrouter/JobRouterServiceVersion.java

```diff
@@ -4,12 +4,10 @@
 
 package com.azure.communication.jobrouter;
 
-import com.azure.core.util.ServiceVersion;
-
 /**
  * Service version of JobRouterClient.
  */
-public enum JobRouterServiceVersion implements ServiceVersion {
+public enum JobRouterServiceVersion implements com.azure.core.util.ServiceVersion {
```

Comment:
We should not have to fully qualify the interface here. We don't expect to have a name conflict for this enum type.